velox/exec/tests/FunctionSignatureBuilderTest.cpp

```diff
@@ -177,3 +177,74 @@ TEST_F(FunctionSignatureBuilderTest, anyInReturn) {
       },
       "Type 'Any' cannot appear in return type");
 }
+
+TEST_F(FunctionSignatureBuilderTest, scalarConstantFlags) {
+  {
+    auto signature = FunctionSignatureBuilder()
+                         .returnType("bigint")
+                         .argumentType("double")
+                         .constantArgumentType("boolean")
+                         .argumentType("bigint")
+                         .build();
+    EXPECT_FALSE(signature->constantArguments().at(0));
+    EXPECT_TRUE(signature->constantArguments().at(1));
+    EXPECT_FALSE(signature->constantArguments().at(2));
+    EXPECT_EQ(
+        "(double,constant boolean,bigint) -> bigint", signature->toString());
+  }
+
+  {
+    auto signature = FunctionSignatureBuilder()
+                         .typeVariable("T")
+                         .returnType("bigint")
+                         .argumentType("double")
+                         .constantArgumentType("T")
+                         .argumentType("bigint")
+                         .constantArgumentType("boolean")
+                         .variableArity()
+                         .build();
+    EXPECT_FALSE(signature->constantArguments().at(0));
+    EXPECT_TRUE(signature->constantArguments().at(1));
+    EXPECT_FALSE(signature->constantArguments().at(2));
+    std::cout << signature->toString() << std::endl;
```

Comment:
Accidental?

Reply:
Removed, sorry not to check and remove it before the push.